java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1584,6 +1586,30 @@ public T type(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Inserts <code>element</code> in the list at <code>key</code> either before or after the <code>
+     * pivot</code> value.
+     *
+     * @see <a href="https://redis.io/commands/linsert/">redis.io</a> for details.
+     * @param key The key of the list.
+     * @param position The relative position to insert into - either {@link InsertPosition#BEFORE} or
+     *     {@link InsertPosition#AFTER} the <code>pivot</code>.
+     * @param pivot The reference value.
+     * @param element The new element to insert.
+     * @return Command Response - The list length after a successful insert operation. If the <code>
+     *     key</code> doesn't exist or <code>-1</code> if the <code>pivot</code> wasn't found, return
+     *     <code>0</code>.
```

Comment:
missing:
If key holds a value that is not a list, an error is returned.

Reply:
We do not document obvious error cases (wrong key type or arguments).